app/configurator/components/add-dataset-dialog.tsx

```diff
@@ -69,6 +69,8 @@ import SvgIcSearch from "@/icons/components/IcSearch";
 import { useLocale } from "@/locales/use-locale";
 import { exhaustiveCheck } from "@/utils/exhaustiveCheck";
 
+import { BetaTag } from "./beta-tag";
```

Comment:
Super-nit: I like the consistency of having the same "style" for imports, maybe we could somehow enforce aliased ones? I think I changed this in my VS Code settings, but maybe there's some configuration option in `settings.json`. Not for this PR, just a thing that I noticed :)

Reply:
I also like consistency :) I think when I auto import, I automatically get aliases, but here it's because I extracted the component, and when extracting the imports gets aliased.

Reply:
Thank you 🙇‍♂️